velox/dwio/dwrf/writer/WriterContext.cpp

```diff
@@ -138,10 +137,15 @@ int64_t WriterContext::availableMemoryReservation() const {
       generalPool_->availableReservation();
 }
 
-void WriterContext::releaseMemoryReservation() {
+int64_t WriterContext::releaseMemoryReservation() {
+  const auto beforeTotalAvailableReservation = availableMemoryReservation();
```

Comment:
Can we just look at the reservation bytes change at the aggregation memory pool? Thanks!